follow-ups/ai-assisted-coding/slides/08_praktyka.md

```diff
@@ -0,0 +1,38 @@
+<!-- .slide: data-background-image="img/brain-explodes.gif" -->
+## To chyba tyle teorii ;-)
+<!-- .element: class="title-on-dark-bg" -->
+
+
+Jak w praktyce możemy korzystać z asystentów?
+
+
+Smutna prawda jest taka, że nie ma rozwiązań darmowych*
+
+note: podobno github oferuje Copilota za darmo studentom oraz kontrybutorom open source
+
+
+Nawet ChatGPT w wersji "darmowej" jest limitowany do konkretnej liczby zapytań dziennie
+
+
+Dobra wiadomość jest taka, że niektórzy producenci asystentów AI oferują darmowy okres próbny, dzięki czemu można przynajmniej sprawdzić, czy faktycznie to się przydaje
+
+
+### Dygresja (acz nie całkiem)
+
+Konto openAI to nie jest to samo, co konto ChatGPT.\
+Rozszerzenia VS Code oferujące ChatGPT zwykle wymagają wklejenia klucza API z openAI.\
+Jeśli masz wykupionego Chata GPT, nie uda Ci się wygenerować klucza API.
+
+
+## Sprawdzamy!
+
+
+### Poproś asystenta o pomoc przy:
+
+- stworzeniu funkcji konwertującej stopnie Celsjusza na Fahrenheita
+- napisaniu prostej funkcji pobierającej dane z API Instagrama

```

Comment:
Moim zdaniem to bardzo trudne pojęcia: dane i API. Nie wiem czy ktoś zrozumie. I z drugiej strony - jak to miałoby być zwizualizowane?

Reply:
Na razie to zostawiam, bo część praktyczna i tak ma zostać przebudowana